Build fails because of a compilation error.

**Environment:**
 - OS: Ubuntu 18.04
 - SDK or header version if building from repo: main

**Describe the Issue**

Build fails with the message:

```
Vulkan-ValidationLayers/layers/./utils/vk_layer_utils.h: In function ‘constexpr VkExtent3D GetEffectiveExtent(const VkImageCreateInfo&, VkImageAspectFlags, uint32_t)’:
Vulkan-ValidationLayers/layers/./utils/vk_layer_utils.h:415:27: error: call to non-constexpr function ‘bool FormatIsMultiplane(VkFormat)’
     if (FormatIsMultiplane(format)) {
```

--- a/BUILD.md
+++ b/BUILD.md
@@ -234,11 +234,7 @@ cmake --build . --config Release
 
 ### Linux Build Requirements
 
-This repository has been built and tested on the two most recent Ubuntu LTS
-versions. Currently, the oldest supported version is Ubuntu 18.04, meaning
-that the minimum officially supported C++17 compiler version is GCC 7.3.0,
-although earlier versions may work. It should be straightforward to adapt this
-repository to other Linux distributions.
+This repository is regularly built and tested on the two most recent Ubuntu LTS versions.
 
 [CMake 3.17.2](https://cmake.org/files/v3.17/cmake-3.17.2-Linux-x86_64.tar.gz) is recommended.
 

